blt/blt.yml

```diff
@@ -26,8 +29,8 @@ modules:
     enable: [acquia_connector, shield]
     uninstall: [devel, config_readonly]
   prod:
-    enable: [acquia_connector, shield, hs_config_readonly]
-    uninstall: [devel]
+    enable: [acquia_connector, shield, hs_config_readonly, bugherdapi]
+    uninstall: [devel, views_ui]
```

Comment:
yay! no views ui in prod.

Reply:
You know they are going to want it though.. it'll probably come back :(